### PR TITLE
Bumping client packages to 1.0.0

### PR DESCRIPTION
**Pull request by ssimic2**
Preparing for 1.0.0 release of client packages.

(Note: Azure packages are not included in this bump)

### CODE REVIEW

**Review by anthony-murphy — APPROVED** [review]

**Comment by anthony-murphy** [comment]
i think breaking.md needs updates too

**Review by skylerjokiel — APPROVED** [review]
We need to move to ~ from ^ as part of this change. We should be aligning on patch

Other wise it looks good.

**Comment by ssimic2** [comment]
> i think breaking.md needs updates too

Good catch!

**Comment by skylerjokiel** [comment]
> > i think breaking.md needs updates too
> 
> Good catch!

We talked about this offline. We are going to keep the ^ schematics for now to keep parity with the existing behavior and enable us to support on minor versions.

File: packages/test/test-version-utils/src/versionUtils.ts [hunk 1]
```diff
@@ -234,5 +234,12 @@ export function getRequestedRange(baseVersion: string, requested?: number | stri
     if (typeof requested === "string") { return requested; }
     const version = new semver.SemVer(baseVersion);
     // ask for prerelease in case we just bumpped the version and haven't release the previous version yet.
+    if (baseVersion === "1.0.0") {
```
Comment (ssimic2) [inline packages/test/test-version-utils/src/versionUtils.ts:237]: To be cleaned up with separate ticket.

**Comment by skylerjokiel** [comment]
Client build is failing due to some transient code-gen issue. Will merge and follow up after.

**Comment by ssimic2** [comment]
> Client build is failing due to some transient code-gen issue. Will merge and follow up after.

Cleaned up here: https://github.com/microsoft/FluidFramework/pull/10549

**Pull request merged**